about/reservation.html

```diff
@@ -0,0 +1,132 @@
+                <span class="navbar-toggler-icon"></span>
+            </button>
+        </div>
+        <div class="collapse navbar-collapse nav-text-padding" id="navbarCollapse">
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/blog">Blog</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/projects">Projects</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/groups">Groups</a>
+                </li>
+                <li class="nav-item active">
+                    <a class="nav-link" href="#">About<span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/contacts">Contact Us</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</nav>
+
+<div id="myCarousel" class="carousel slide" data-ride="carousel">
+    <div class="carousel-inner">
+        <div class="carousel-item active" style="background-image: url(https://i.loli.net/2018/01/10/5a55c79cd8651.jpg);">
+            <img class="first-slide" src="https://i.loli.net/2018/01/10/5a55c79cd8651.jpg" alt="First slide">
+            <div class="container">
+                <div class="carousel-caption carousel-caption-bright d-none d-block text-left">
+                    <h1>Join Us</h1>
+                    <p>What you need to do &amp; how to do it.</p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<!-- Marketing messaging and featurettes
+================================================== -->
+<!-- Wrap the rest of the page in another container to center all the content. -->
+
+<div class="container marketing projects">
+
+    <div>
+        <h1>Basic Process</h1>
+        <hr>
+        <div class="alert alert-primary" role="alert">
+            <h4 class="alert-heading"><i class="fa fa-lightbulb-o" aria-hidden="true"></i> Step 1 · Make a decision</h4>
+            <p>We do not have a fixed time for recruiting new members (But usually during the students in school), if you are interested in UIC HCC Computer Society, you can contact us immediately.</p>
+            <hr>
+            <p class="mb-0">
+                <a class="btn btn-primary" href="/" role="button">Know More about UIC HCC Computer Society</a>
+            </p>
+        </div>
+        <div class="alert alert-warning" role="alert">
+            <h4 class="alert-heading"><i class="fa fa-calendar" aria-hidden="true"></i> Step 2 · Make a reservation</h4>
+            <p>If you want to join us, the first thing you should do is confirm the interview time with us by email.</p>
```

Comment:
~~If you want to join us, t~~ **T**he first thing you ~~should~~ **need to** do is **to** confirm the ~~interview~~ time **of interview** with us by email.